spec/formats/news_story_spec.rb

```diff
@@ -52,19 +58,39 @@ def then_i_can_publish_the_document
 
   def content_body
     {
-      "links" => {
+      "links" => a_hash_including(
         "topical_events" => [linkable["content_id"]],
         "worldwide_organisations" => [linkable["content_id"]],
         "world_locations" => [linkable["content_id"]],
         "organisations" => [linkable["content_id"]],
         "primary_publishing_organisation" => [linkable["content_id"]],
-      },
+        "roles" => [role_content_id],
+        "people" => [person_content_id],
+      ),
       "title" => "A great title",
       "document_type" => "news_story",
       "description" => "A great summary",
     }
   end
 
+  def role_appointment_links
```

Comment:
Could make this a little shorter by memoising here and merging `role_appointment_links["links"]` into `content_body["links"]`, which would reduce duplication in the two format specs.

Reply:
I don't think we can (or I've not understood you correctly) as they have different link names. singular role/person for a role_appointment, and plural roles/people for news

Reply:
Oh yeah. Alternatively you could still memoize the role_appointment_links and extract out the content_ids in the content body - just thought you could save on a couple of methods.